src/Search/Provider/SQLProvider.php

```diff
@@ -5269,7 +5269,21 @@ public static function giveItem(
 
                 case 'glpi_ticketsatisfactions.satisfaction':
                     if ($html_output) {
-                        return \TicketSatisfaction::displaySatisfaction($data[$ID][0]['name']);
+                        $ticket = Ticket::getById($data['raw']['id']);
```

Comment:
Is it possible to get entity id from search option additionnal fields? With your solution, if you display many tickets, it will produce many additionnal queries.

Reply:
I'm not sure it can be done with the `additionalfields` parameter as we need the entities_id field from the ticket/change table, not the satisfaction table

```diff
diff --git a/src/CommonITILSatisfaction.php b/src/CommonITILSatisfaction.php
index 5c41f84940..212e05343e 100644
--- a/src/CommonITILSatisfaction.php
+++ b/src/CommonITILSatisfaction.php
@@ -412,7 +412,8 @@ abstract class CommonITILSatisfaction extends CommonDBTM
             'massiveaction'      => false,
             'joinparams'         => [
                 'jointype'           => 'child'
-            ]
+            ],
+            'additionalfields'   => ['entities_id'],
         ];

         $tab[] = [
(
```

![image](https://user-images.githubusercontent.com/42734840/203027155-be25eae7-fe1d-49bf-97fe-faac26f2410b.png)